src/nvim/os/shell.c

```diff
@@ -37,6 +37,40 @@ typedef struct {
 #endif
 
 
+/// Process command string with 'shellxescape' (p_sxe) and 'shellxquote'
+/// (p_sxq)
+///
+/// @param cmd Command string
+/// @return A newly allocated command string. It must be freed with
+///         `xfree` when no longer needed.
+static char *cmd_sxe_sxq(const char *cmd)
```

Comment:
I think this function name is very nondescriptive.. maybe something like `shell_escape`?